parser_test.go

```diff
@@ -139,7 +139,15 @@ const exampleStateFile = `
 
 const expectedListOutput = `
 {
-	"all":	 {"datacenter": "mydc", "olddatacenter": "<0.7_format", "ids": [1, 2, 3, 4], "map": {"key": "value"}},
+	"all":	 {
+		"hosts": ["10.0.0.1"], 
```

Comment:
It would be nice to update the test cases to test the unique host filtering functionality.